api-server/src/server/middlewares/rate-limit.js

```diff
@@ -11,7 +11,11 @@ export default function rateLimitMiddleware() {
     max: 10,
     standardHeaders: true,
     legacyHeaders: false,
+    keyGenerator: req => {
+      return req.headers['x-forwarded-for'] || 'localhost';
+    },
```

Comment:
I am not 100 convinced on this and I think this could still cause issues for users who share a IP address.